src/containers/Home/Jobs/style.scss

```diff
@@ -0,0 +1,46 @@
+.job-active{
+    border: solid 1px red;
+}
+.job{
+    margin-left: 20%;
+    margin-right: 20%;
+    padding: 40px;
+    position: relative;
+    .job__tittle{
```

Comment:
you can use `&__tittle` for the shorter syntax

Reply:
i've changed